jdbc-adapter/virtualschema-jdbc-adapter/src/main/java/com/exasol/adapter/dialects/hive/HiveSqlGenerationVisitor.java

```diff
@@ -124,72 +130,60 @@ public String visit(final SqlPredicateLikeRegexp function) throws AdapterExcepti
     public String visit(final SqlFunctionScalar function) throws AdapterException {
         String sql = super.visit(function);
```

Comment:
Replace by returns in all paths. Move `super` call to default.

Reply:
done